java/challenges/lib/src/test/java/challenges/BinarySearchTest.java

```diff
@@ -0,0 +1,21 @@
+package challenges;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class BinarySearchTest {
+
+  @Test
```

Comment:
Good work on the test, I'd suggest writing a few more in the future. What if the array was empty? What if the key didn't exist in the array?